app/main.py

```diff
@@ -2,0 +1,10 @@
+from shutil import copy
+
+
+def copy_file(command: str) -> None:
+    _, source_file, destination_file = command.split(" ")
+
+    if source_file == destination_file:
+        return
+
+    copy(source_file, destination_file)
```

Comment:
Code Efficiency Issue: There is no error handling for the copy operation. What if the source file does not exist, or the destination directory is not writable? Consider adding a try-except block to handle potential errors.